README.md

```diff
@@ -128,10 +148,14 @@ class AbstractDatabaseConnection {
 
 to configure the maximal reasonable length consider using the following parameter:
 ```xml
-<configuration>
-  <maxClassNameLen>15</maxClassNameLen>
+<plugin>
+  <groupId>ru.l3r8y</groupId>
+  <artifactId>oop-cop</artifactId>
   ...
-</configuration>
+  <configuration>
+    <maxClassNameLen>15</maxClassNameLen>
```

Comment:
you can add a comment that this is a default value that does not need to be specified

Reply:
you mean `13`?

Reply:
> you mean `13`?

sure